setup.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS responses (
+  id    SERIAL PRIMARY KEY,
+  added_date    timestamp WITH time zone,
+  survey_response    jsonb
+);
```

Comment:
I've added indexes in https://github.com/ONSdigital/sdx-store/commit/26453fde416ea6e99171504f64f5ca4448057e85